PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -58,12 +72,76 @@ protected void configureShell(Shell newShell) {
 		newShell.setText("Link New HDInsight Cluster");
 	}
 
+	// we cannot just override method 'helpPressed' since the method is defined as private in the parent class 'TrayDialog'
+	// so we disable the default help button and mock a button with different usage
+	protected void helpPressed() {
+		try {
+			PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL("https://go.microsoft.com/fwlink/?linkid=866472"));
+		} catch (PartInitException | MalformedURLException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
+	
+	private ToolBar createHelpImageButton(Composite parent, Image image) {
+        ToolBar toolBar = new ToolBar(parent, SWT.FLAT | SWT.NO_FOCUS);
+        ((GridLayout) parent.getLayout()).numColumns++;
+		toolBar.setLayoutData(new GridData(GridData.HORIZONTAL_ALIGN_CENTER));
```

Comment:
Please fix the indents to spaces instead of tabs.

Reply:
replace tab with four spaces in the new commit.